tsl/test/expected/dist_compression.out

```diff
@@ -880,3 +880,98 @@ SELECT * from test_recomp_int_chunk_status ORDER BY 1;
  _dist_hyper_4_14_chunk |            1
 (1 row)
 
+---run copy tests
+--copy data into existing chunk + for a new chunk
+COPY test_recomp_int  FROM STDIN WITH DELIMITER ',';
+SELECT time_bucket(20, time ), count(*)
+FROM test_recomp_int
+GROUP BY time_bucket( 20, time) ORDER BY 1;
+ time_bucket | count 
+-------------+-------
+           0 |    12
+         100 |     3
+(2 rows)
+
+--another new chunk
+INSERT INTO test_recomp_int VALUES( 65, 10);
+SELECT * from test_recomp_int_chunk_status ORDER BY 1;
+       chunk_name       | chunk_status 
+------------------------+--------------
+ _dist_hyper_4_14_chunk |            3
+ _dist_hyper_4_15_chunk |            0
+ _dist_hyper_4_16_chunk |            0
+(3 rows)
+
+--compress all 3 chunks ---
+SELECT compress_chunk(chunk, true)
+FROM show_chunks('test_recomp_int') AS chunk
+ORDER BY chunk;
+                compress_chunk                
+----------------------------------------------
+ _timescaledb_internal._dist_hyper_4_14_chunk
+ _timescaledb_internal._dist_hyper_4_15_chunk
+ _timescaledb_internal._dist_hyper_4_16_chunk
+(3 rows)
+
+--interleave copy into 3 different chunks and check status--
```

Comment:
Shouldn't status be checked immediately after compression above rather than after COPY?

Reply:
added a check after call to compress_chunk as well